### DISCUSSION TITLE
Explain extension VSCODE doesn't take the text

### DISCUSSION THREAD

**Discussion by gugudwt** [community/community#118675] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi,

When i select my code and press CMD + I, I select the command /explain and then write my text.
VSCode Copilot chat extension take the command and the contextual file, but not the text i write.

<img width="451" alt="Screenshot 2024-04-11 alle 11 17 40" src="https://github.com/community/community/assets/13451968/838fc203-6189-476f-a7cd-19e8802f47a8">
<img width="297" alt="Screenshot 2024-04-11 alle 11 17 46" src="https://github.com/community/community/assets/13451968/b553395a-603f-4a78-b788-13497aaef9ea">

**Comment by jfullstackdev** [comment]


**Reply by gugudwt** [reply]
I don't think your answer is helpful

**Comment by eymeen** [comment]
hey @gugudwt 
if this issue started appearing recently and before it was working fine, try to create a new vscode profile OR use vscode insiders and only download the explain extension and test it. If it works, then you have to re-install it on your main profile, if it still has the problem, this means there's another extension that affecting its functionality, and probably it would be text-controlling extensions like snippets/code hints... (anything changes the original way of vs code writing). 

I hope it helps

**Reply by gugudwt** [reply]
Hi,

thank you for the answer.

I tried to create another profile but the problem persist.
I uninstalled e reinstalled vscode, without syncronization, and the problem persist.

The problem started three/four days ago.

Maybe something that does not depend on vscode / extensions but on the machine configuration?

Which kind of debug can i do?

**Reply by eymeen** [reply]
well it seems like you're not the only one check #118670 , I think you need to contact copilot customer service for that because it seems it's a problem in their side

**Reply by gugudwt** [reply]
Where i can contact them?

**Reply by eymeen** [reply]
I think it's only available for those who have copilot subscription, but you can contact github about it visiting [this page](https://support.github.com/request) as the maintainers said in #40816

**Comment by mjmaurer** [comment]
I'm having the same issue. It's an issue with the pre-release version as well

**Comment by stevensabharwal** [comment]
Any additional info on this? As of this week myself and my peers are now experiencing the same issue beginning at the same time.

**Reply by gugudwt** [reply]
To date I still haven't been able to solve it
